src/gui/skin_gothic.cpp

```diff
@@ -36,42 +30,70 @@ namespace REGoth
     bs::HFont fontGothicOld10WhiteHi = loadFont("FONT_OLD_10_WHITE_HI.FNT");
-    bs::HFont fontGothicOld20White = loadFont("FONT_OLD_20_WHITE.FNT");
+    bs::HFont fontGothicOld20White   = loadFont("FONT_OLD_20_WHITE.FNT");
     bs::HFont fontGothicOld20WhiteHi = loadFont("FONT_OLD_20_WHITE_HI.FNT");
 
     bs::GUIElementStyle baseStyle;
-    baseStyle.font = fontGothicDefault;
-    baseStyle.fontSize = 18;
-    baseStyle.wordWrap = true;
+    baseStyle.font             = fontGothicDefault;
+    baseStyle.fontSize         = 17;
+    baseStyle.wordWrap         = true;
+    baseStyle.normal.textColor = bs::Color::White;
     skin->setStyle("Label", baseStyle);
 
     bs::GUIElementStyle labelDefault = baseStyle;
-    labelDefault.font = fontGothicOld10White;
-    labelDefault.fontSize = 18;
+    labelDefault.font                = fontGothicOld10White;
+    labelDefault.fontSize            = 17;
     skin->setStyle("GothicLabel", labelDefault);
 
     bs::GUIElementStyle labelLarge = baseStyle;
-    labelLarge.font = fontGothicOld20White;
-    labelLarge.fontSize = 35;
+    labelLarge.font                = fontGothicOld20White;
+    labelLarge.fontSize            = 36;
     skin->setStyle("GothicLabelLarge", labelLarge);
 
     bs::GUIElementStyle labelDefaultHighlighted = baseStyle;
-    labelDefaultHighlighted.font = fontGothicOld10WhiteHi;
-    labelDefaultHighlighted.fontSize = 18;
+    labelDefaultHighlighted.font                = fontGothicOld10WhiteHi;
+    labelDefaultHighlighted.fontSize            = 17;
     skin->setStyle("GothicLabelHighlighted", labelDefaultHighlighted);
 
     bs::GUIElementStyle labelLargeHighlighted = baseStyle;
-    labelLargeHighlighted.font = fontGothicOld20WhiteHi;
-    labelLargeHighlighted.fontSize = 35;
+    labelLargeHighlighted.font                = fontGothicOld20WhiteHi;
+    labelLargeHighlighted.fontSize            = 36;
     skin->setStyle("GothicLabelLargeHighlighted", labelLargeHighlighted);
 
-    return skin;
+    bs::GUIElementStyle labelObjectFocus = baseStyle;
+    labelObjectFocus.font                = fontGothicOld10WhiteHi;
+    labelObjectFocus.fontSize            = 17;
+    labelObjectFocus.wordWrap            = false;
+    // labelObjectFocus.textHorzAlign = bs::TextHorzAlign::THA_Center;
+    // labelObjectFocus.textVertAlign = bs::TextVertAlign::TVA_Center;
+    skin->setStyle("GothicLabelObjectFocus", labelObjectFocus);
+
+    bs::GUIElementStyle subtitleBox = baseStyle;
+    subtitleBox.normal.texture      = gOriginalGameResources().sprite("DLG_CHOICE.TGA");
+    skin->setStyle("GothicDialogueBoxBackground", labelLargeHighlighted);
+
+    skin->setStyle("GothicSubtitleBoxCharacterName", labelDefaultHighlighted);
+    skin->setStyle("GothicSubtitleBoxText", labelDefault);
+
+    // skin->setStyle("Button", const GUIElementStyle &style)
+
+    // Cache the skin for later use
+    s_SkinGothic = skin;
+
+    return bs::BuiltinResources::instance().getGUISkin();
```

Comment:
Does this not get rid of all the ElementStyle definitions we do here and just returns the skin provided by bsf?

Reply:
Yes, my custom skin doesn't really work yet :smile:

Reply:
I would suggest adding our styles on to the built-in skins and returning that. 
Best of both worlds :P

Reply:
Done that.